api/WindowClient.json

```
@@ -180,7 +180,7 @@
               "version_added": "36"
             },
             "safari": {
-              "version_added": "11.1"
+              "version_added": false
```

Because the method exists, we have to use the `partial_implementation` flag instead of marking it as unsupported (see [this guideline](https://github.com/mdn/browser-compat-data/blob/main/docs/data-guidelines.md#non-functional-defined-names-imply-partial_implementation) for background information). How about this instead:

```suggestion
              "version_added": "11.1",
              "partial_implementation": true,
              "notes": "This method exists, but always throws <code>NotSupportedError</code>."
```